weblab/experiments/views.py

```diff
@@ -45,10 +47,33 @@ class ExperimentsView(TemplateView):
     template_name = 'experiments/experiments.html'
 
 
+class ExperimentTasks(LoginRequiredMixin, ListView):
+    """
+    Show running and queued versions of all experiments
+    Delete checked versions
+    """
+    model = Experiment
+    template_name = "experiments/experiment_tasks.html"
+
+    def get_queryset(self):
+        return self.model.objects.filter(
+            Q(author=self.request.user) &
+            Q(versions__status=ExperimentVersion.STATUS_QUEUED) |
```

Comment:
Check precedence here - you might need extra brackets! (Probably better to have extra brackets for clarity anyway.)

Reply:
as suggested changed query to use running experiments